packages/salesforcedx-vscode-apex/src/index.ts

```diff
@@ -12,23 +12,84 @@ import {
   DEBUGGER_LINE_BREAKPOINTS
 } from './constants';
 import * as languageServer from './languageServer';
+import { ApexTestOutlineProvider } from './views/testOutline';
+
+export type ApexTestRequestInfo = {
```

Comment:
TODO: Need to reconcile this with the changes that are happening on the jorje side.